pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -858,7 +859,7 @@ func (k *Kubernetes) ConfigVolumes(name string, service kobject.ServiceConfig) (
 					}
 				}
 
-				createdPVC, err := k.CreatePVC(volumeName, volume.Mode, defaultSize, volume.SelectorValue)
+				createdPVC, err := k.CreatePVC(volumeName, volume.Mode, defaultSize, volume.SelectorValue, volume.StorageClassName)
```

Comment:
Missing docs update ,unit test and integreated test. Please check the github ci workflow

Reply:
add test file